lib/filesystem/CZipSaver.cpp

```diff
@@ -72,7 +72,7 @@ CZipOutputStream::~CZipOutputStream()
 
 si64 CZipOutputStream::write(const ui8 * data, si64 size)
 {
-	int ret = zipWriteInFileInZip(handle, (const void*)data, (unsigned)size);
+	int ret = zipWriteInFileInZip(handle, (const void *)data, static_cast<unsigned>(size));
```

Comment:
`const ui8 *` should be castable to `const void*` without explicit casts
```suggestion
	int ret = zipWriteInFileInZip(handle, data, static_cast<unsigned>(size));
```

Reply:
done, let's look to CI.